cishouseholds/derive.py

```diff
@@ -872,3 +872,38 @@ def assign_correct_age_at_date(df: DataFrame, column_name_to_assign, reference_d
         + F.round((F.col("month_more") + F.col("day_more")) / 3, 0).cast("int"),
     )
     return df.drop("month_more", "day_more")
+
+
+def assign_days_since_think_had_covid_group(
```

Comment:
Change function name as applicable to more than one variable

assign_days_since_grouped_variable